### DISCUSSION TITLE
Welcome to anything-minecraft Discussions!

### DISCUSSION THREAD

**Discussion by JustDoom** [Anything-Minecraft-Team/anything-minecraft#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JustDoom** [comment]
hello!

**Comment by Kruemmelspalter** [comment]
@JustDoom you should maybe give someone else write permissions for the repo so you don't have like 20 new PRs or issues in the (your) morning

**Comment by ghost** [comment]
hello

**Comment by JustDoom** [comment]
lol, forgot about this
